usaspending_api/search/tests/integration/spending_by_category/test_spending_by_awarding_agency.py

```diff
@@ -79,11 +60,36 @@ def test_correct_response(client, monkeypatch, elasticsearch_transaction_index,
         "limit": 10,
         "page_metadata": {"page": 1, "next": None, "previous": None, "hasNext": False, "hasPrevious": False},
         "results": [
-            {"amount": 10.0, "name": "Awarding Toptier Agency 3", "code": "TA3", "id": 1005},
```

Comment:
The purpose of this test is to make sure that we are correctly grabbing the Toptier Agency ID by using the `is_toptier` flag. If the new use of fixtures resulted in this change then it is no longer fulfilling its original intent. The fixture should be updated to match the original relations so that this test has the same purpose as before.

Reply:
The ID changed because the agencies now come from a helper function that picks ids in a more standard way, but agency 1003 (defined in line 24 of the fixture class) is the same structure as the old agency 1005 (defined in line 23 of the old fixture class). The new helper method just standardizes that the `agency` table entry for the toptier ends with the same number as the `toptier_agency` table entry, whereas the old code set it up so that the agency row for the toptier always ended with 5, while subtiers ended with the same number as the agency row.

Reply:
The original structure had transaction_normalzied record with an awarding_agency_id of 1003 (which is the same as the new structure). However, the difference is that in the old the `is_toptier` flag is set to False and in the new it is set to True. That distinction is important since it checks that the code is looking up the correct awarding_toptier_agency_id of 1005 (where the `is_toptier` flag is set to True). The new structure doesn't perform that check since 1003 is set to True and 1005 is set to False, meaning no need to perform that check.

Reply:
Doesn't that still make the code perform the check, just with 1003 returning true and 1005 returning false?